[protobuf] Unable to install

```
mawk: run time error: pipe execution not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
make[4]: *** [Makefile:7704: test-suite.log] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/v3.3.0.tar.gz.dir/protobuf-3.3.0/src'
make[3]: *** [Makefile:7812: check-TESTS] Error 2
make[3]: Leaving directory '/usr/local/tmp/crew/v3.3.0.tar.gz.dir/protobuf-3.3.0/src'
make[2]: *** [Makefile:7925: check-am] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/v3.3.0.tar.gz.dir/protobuf-3.3.0/src'
make[1]: *** [Makefile:7927: check] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/v3.3.0.tar.gz.dir/protobuf-3.3.0/src'
make: *** [Makefile:1406: check-recursive] Error 1
protobuf failed to build: `make check` exited with 2
```

--- a/packages/doxygen.rb
+++ b/packages/doxygen.rb
@@ -8,9 +8,17 @@ class Doxygen < Package
   source_sha256 '18bc3790b4d5f4d57cb8ee0a77dd63a52518f3f70d7fdff868a7ce7961a6edc3'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/doxygen-1.8.14-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/doxygen-1.8.14-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/doxygen-1.8.14-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/doxygen-1.8.14-0-chromeos-x86_64.tar.xz',
 
   })
   binary_sha256 ({
+    aarch64: '7b9c085e0b949dc392fbef741fffb1990ca5c4e4da9574e70bc78f404c4f6823',
+     armv7l: '7b9c085e0b949dc392fbef741fffb1990ca5c4e4da9574e70bc78f404c4f6823',
+       i686: 'ffdefe84883879da73bba23ea6aed9cf652efa868b384910b0830afcd5028b15',
+     x86_64: 'd6627ee0caad73c73f622d49c4a2702680d66bc231ef1d42acf9f61fa0949644',
     
   })
 

--- a/packages/font_util.rb
+++ b/packages/font_util.rb
@@ -7,6 +7,19 @@ class Font_util < Package
   source_url 'https://www.x.org/archive/individual/font/font-util-1.3.1.tar.bz2'
   source_sha256 'aa7ebdb0715106dd255082f2310dbaa2cd7e225957c2a77d719720c7cc92b921'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/font_util-1.3.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/font_util-1.3.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/font_util-1.3.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/font_util-1.3.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'f4451fb18696cf52f4ae32c883b48c4f3d01498d86cf8de2bd23ad413955e8c4',
+     armv7l: 'f4451fb18696cf52f4ae32c883b48c4f3d01498d86cf8de2bd23ad413955e8c4',
+       i686: '8938497e0efbd1ea684742433177fd0d76d818e31b583d1c903a0b45935c5bcf',
+     x86_64: '8bac57c788ba219f59274697ce0b056d61232ec78db7ed7e110cf7917896d1e6',
+  })
+
     depends_on 'util_macros'
 
     

--- a/packages/fribidi.rb
+++ b/packages/fribidi.rb
@@ -7,8 +7,21 @@ class Fribidi < Package
   source_url 'https://github.com/fribidi/fribidi/releases/download/v1.0.1/fribidi-1.0.1.tar.bz2'
   source_sha256 'c1b182d70590b6cdb5545bab8149de33b966800f27f2d9365c68917ed5a174e4'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/fribidi-1.0.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/fribidi-1.0.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/fribidi-1.0.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/fribidi-1.0.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd2c2acf788e0fc1c3d5330ead4bf2f184c7667847c51fe6e8da183cba9e77748',
+     armv7l: 'd2c2acf788e0fc1c3d5330ead4bf2f184c7667847c51fe6e8da183cba9e77748',
+       i686: '0b0515f77ac80bfd3cf9332b3bc5980e04e3664e9b2f5957f4f12ba48f9dd02a',
+     x86_64: 'f408ec87bb6df6e4d375ba9b90d5d79cbdef7180e6fb69bdf31610756ae31f7d',
+  })
+
   def self.build
-    system "./configure"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 

--- a/packages/libevdev.rb
+++ b/packages/libevdev.rb
@@ -7,6 +7,19 @@ class Libevdev < Package
   source_url 'https://www.freedesktop.org/software/libevdev/libevdev-1.5.9.tar.xz'
   source_sha256 'e1663751443bed9d3e76a4fe2caf6fa866a79705d91cacad815c04e706198a75'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libevdev-1.5.9-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libevdev-1.5.9-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libevdev-1.5.9-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libevdev-1.5.9-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'ee50da24a54da6574d50571871052104ed46256280f4bc7c5698e76d7ecf4fd3',
+     armv7l: 'ee50da24a54da6574d50571871052104ed46256280f4bc7c5698e76d7ecf4fd3',
+       i686: 'af2583a717277459bd65c27b994a36ef2fc1e018278773c831153abc871e567d',
+     x86_64: '6f113678d99a5f4fc52db49702bb209e845de505c1bb456b4f2d1c862e9ff7ed',
+  })
+
   depends_on 'doxygen'
   depends_on 'python27'
   

--- a/packages/protobuf.rb
+++ b/packages/protobuf.rb
@@ -8,8 +8,16 @@ class Protobuf < Package
   source_sha256 '826425182ee43990731217b917c5c3ea7190cfda141af4869e6d4ad9085a740f'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/protobuf-3.5.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/protobuf-3.5.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/protobuf-3.5.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/protobuf-3.5.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'c9f90e5821643761acf7aabb0e60a9d2d295071b1f3a06664e78ff071483edb3',
+     armv7l: 'c9f90e5821643761acf7aabb0e60a9d2d295071b1f3a06664e78ff071483edb3',
+       i686: '9df9e1d3deb47a836193f72385eaf383fd2a49e0246e0259fea856a59290097c',
+     x86_64: '5da67b574e7df6c9e58b2bcee98b3b78cf7c95d6458350a3a1036981b0ad4f99',
   })
 
   depends_on 'unzip' => :build

